docs/How_To/Tutorials/blog_writer.rst

```diff
@@ -109,7 +109,7 @@ Step 2. Run:
 
 .. code:: bash
 
-   python main.py --config agent_config.yml --transcript transcript.txt
+   python main.py --config agent_config.yml
```

Comment:
I say we keep the "transcript" argument and allow the user to specify which file in the Transcripts folder they want to use. Then instead of always naming the JSON `blueprint.json` and the final output `blog.md`, we name them `blueprint_[input_filename].json` and 'blog_[input_filname].md` respectively. The less hardcoding the better!

Reply:
changes done in this commit 
https://github.com/andytai7/sherpa/pull/1/commits/a9c3d7ee28d898dc8fda7a5118d4e53798e070a6